civis/ml/_model.py

```diff
@@ -151,6 +151,26 @@ def _load_estimator(job_id, run_id, filename='estimator.pkl', client=None):
         return joblib.load(path)
 
 
+def _exception_from_logs(job_id, run_id, client, nlog=15):
+    """Create an exception if the log has a recognizable error
+
+    Search "error" emits in the last ``n_log`` lines.
+    This function presently recognizes the following errors:
+
+    - MemoryError
+    """
+    logs = client.scripts.list_containers_runs_logs(job_id, run_id, limit=nlog)
+    msgs = [l['message'] for l in logs if l['level'] == 'error']
+
+    # Check for memory errors
+    mem_err = [m for m in msgs if m.startswith('Process ran out of its')]
```

Comment:
I don't think we should be scraping logs for errors.  Can we create a more general error (`CivisModelTrainError`?) and show the last n lines of the log as the message instead?